common4j/src/main/com/microsoft/identity/common/java/interfaces/INameValueStorage.java

```diff
@@ -55,4 +68,10 @@
      * Clear all data from the storage.
      */
     void clear();
+
+    /**
+     * Get all keys in this storage.
```

Comment:
Interesting...why do we need this? Why not have a getAll method instead that returns values?

Reply:
to iterate and dump the data as-is to the databag/bundle.